examples/allennlp/allennlp_simple.py

```diff
@@ -19,59 +19,66 @@
 """
 
 import os
+import random
 import shutil
 
 import allennlp
 import allennlp.data
 import allennlp.models
 import allennlp.modules
+import numpy
 import torch
 
 import optuna
+from optuna.integration import AllenNLPPruningCallback
 
 
```

Comment:
I think we can add a version check here since this example raises errors with `AllenNLP < 1.0.0`.

Reply:
We can check the version, for example, as follows:
```python
import pkg_resources

if pkg_resources.parse_version(allennlp.__version__) < pkg_resources.parse_version("1.0.0"):
    raise RuntimeError("AllenNLP>=1.0.0 is required for this example.")

```